src/qbProxy.js

```diff
@@ -147,6 +147,13 @@ ServiceProxy.prototype = {
          * Private functions
          * Only for ServiceProxy.ajax() method closure
          */
+
+        function fixedEncodeURIComponent(str) {
```

Comment:
Where this function is used?

Reply:
https://github.com/QuickBlox/quickblox-javascript-sdk/pull/319/files#diff-cecc753aa99688e1f8c36cb3d501513fR176

In `_getBodyRequest` to encode key and value;